packages/chain-adapters/package.json

```diff
@@ -37,19 +37,19 @@
     "@shapeshiftoss/caip": "^1.5.1",
     "@shapeshiftoss/hdwallet-core": "^1.18.2",
     "@shapeshiftoss/hdwallet-native": "^1.18.2",
-    "@shapeshiftoss/types": "^1.13.1",
-    "@shapeshiftoss/unchained-client": "^4.0.0",
-    "@shapeshiftoss/unchained-tx-parser": "^4.0.0",
+    "@shapeshiftoss/types": "^1.19.1",
```

Comment:
Looks like types package is at `1.20.0` on main if we want to use most recent. Don't think this version matters though by itself as it is a peer dependency. Only version that would affect chain adapters itself would be the `devDependencies` version number. (That is my understanding at least...)